stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -159,7 +159,10 @@ internal class ChatDomainImpl internal constructor(
     override val errorEvents: LiveData<Event<ChatError>> = _errorEvent
 
     /** the event subscription, cancel using repo.stopListening */
-    private var eventSubscription: Disposable? = null
+    private var eventSubscription: Disposable = object : Disposable {
+        override val isDisposed: Boolean = true
+        override fun dispose() { }
+    }
```

Comment:
It would be nice to move this `Disposable` implementation into a "static" field and give it a descriptive name